doc/content/api/node_pipelines.md

```diff
@@ -21,12 +21,21 @@ Run an itk.js Emscripten module with Node.js.
 <dl>
   <dt>`IOTypes.Text`</dt><dd>A UTF8-encoded string.</dd>
   <dt>`IOTypes.Binary`</dt><dd>A binary [`Uint8Array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array).</dd>
-  <dt>`IOTypes.Image`</dt><dd>An [`itk/Image`](./Image.html).</dd>
-  <dt>`IOTypes.Mesh`</dt><dd>An [`itk/Mesh`](./Mesh.html).</dd>
+  <dt>`IOTypes.Image`</dt><dd>An [`itk/Image`](./Image.html). To generate this data type in C++, use the [`itk::ImageFileWriter`](https://itk.org/Doxygen/html/classitk_1_1ImageFileWriter.html), add `BridgeJavaScript` module to the ITK `find_package` `COMPONENTS` in your *CMakeLists.txt*, and use `.json` for the image file name extension.</dd>
```

Comment:
The `inline code formatting` (with back-ticks around) does not seem to play weel with the `<dl><dt>` html tags, see picture below:

<img width="500" alt="screen shot 2019-02-06 at 11 21 49 am" src="https://user-images.githubusercontent.com/4910855/52356319-fbd6c100-2a01-11e9-8e37-e5ea6a21d934.png">

Reply:
It seems the issue applies to the hyperlinks also.

Reply:
Interesting -- I do not observe this locally (markdown parser version issue?) but it does ring a bell from the past -- correcting along with the `the the` issue.

Reply:
Doesn't seem resolved, was it?
https://github.com/thewtex/itk-js/blob/48c44b44415711cd8de46f06e772dc22f97e03b5/doc/content/api/node_pipelines.md